native/storybook/storybook.js

```diff
@@ -76,12 +76,12 @@ configure(() => {
   require('../packages/react-native-bpk-component-spinner/stories');
   require('../packages/react-native-bpk-component-star-rating/stories');
   require('../packages/react-native-bpk-component-switch/stories');
-  require('../packages/react-native-bpk-component-text/stories');
   require('../packages/react-native-bpk-component-text-input/stories');
-  require('../packages/react-native-bpk-component-touchable-overlay/stories');
+  require('../packages/react-native-bpk-component-text/stories');
   require('../packages/react-native-bpk-component-touchable-native-feedback/stories');
-  require('../packages/react-native-bpk-theming/stories');
+  require('../packages/react-native-bpk-component-touchable-overlay/stories');
   require('../packages/react-native-bpk-styles/stories');
+  require('../packages/react-native-bpk-theming/stories');
```

Comment:
This file was reordered slightly by the sort function implemented in create-component.js. We think this is probably fine.